tools/downloader/common.py

```diff
@@ -454,18 +454,19 @@ def apply(self, reporter, output_dir):
 Postproc.types['unpack_archive'] = PostprocUnpackArchive
 
 class Model:
-    def __init__(self, name, subdirectory, files, postprocessing, mo_args, quantizable, framework,
+    def __init__(self, name, subdirectory, files, postprocessing, mo_args, quantized, quantizable, framework,
                  description, license_url, precisions, task_type, conversion_to_onnx_args):
         self.name = name
         self.subdirectory = subdirectory
         self.files = files
         self.postprocessing = postprocessing
         self.mo_args = mo_args
+        self.quantized = quantized
```

Comment:
I don't think this field needs to be saved; nothing uses it afterwards.

Reply:
removed